shell/linters/prettier.sh

```diff
@@ -7,16 +7,16 @@ source "$DIR/languages/nodejs.sh"
 
 # Why: Used by the script that calls us
 # shellcheck disable=SC2034
-extensions=(yaml yml json md)
+extensions=(yaml yml json md ts)
 
 prettier_linter() {
   yarn_install_if_needed >/dev/null
-  git ls-files '*.yaml' '*.yml' '*.json' '*.md' | xargs -n40 "node_modules/.bin/prettier" -l --loglevel warn
+  git ls-files '*.yaml' '*.yml' '*.json' '*.md' '*.ts' | xargs -n40 "node_modules/.bin/prettier" -l --loglevel log || exit $?
```

Comment:
Adding an `exit` here seems risky. Is this part of the linter contract that we were forgetting?

Usually you'd want to return a value that the caller could check ...

Reply:
You're looking for "return" here instead

Reply:
we can use 'return' instead of 'exit'